GrocifyApp.UI/src/pages/Settings.jsx

```diff
@@ -115,14 +140,54 @@ function Settings(props) {
                         <span className="slider"></span>
                     </label>
                 ) : (
-                    <div className="icon--w16 cursor-pointer">
+                    <div className="icon cursor-pointer">
                         <ReactSVG className={"react-svg " + settingItem.color} src={settingItem.icon == null ? ChevronIcon : settingItem.icon} />
                     </div>
                 )}
             </div>
         );
     }
 
+    const settingsItems = [
```

Comment:
you can move, the problem is with the link, you put link: openModal, but you must put [link](link: () => openModal()). because in the first option, it will run openModal at the beggining, but if we use () => , openModal is only called when clicking